spec/pubid_iso/identifier_spec.rb

```diff
@@ -249,4 +249,13 @@
     it_behaves_like "converts pubid to urn"
     it_behaves_like "converts pubid to pubid"
   end
+
+  context "GUIDE ISO/CEI 71:2001(F)" do
+    let(:original) { "GUIDE ISO/CEI 71:2001(F)" }
+    let(:pubid) { "ISO/IEC Guide 71:2001(fr)" }
+    let(:urn) { "urn:iso:std:iso-iec:guide:71:fr" }
+
+    it_behaves_like "converts pubid to urn"
+    it_behaves_like "converts pubid to pubid"
+  end
```

Comment:
Also need to generate the French PubID.

There are two differences to account for:

* "Guide" appears as a prefix to the publisher in French but as a suffix to the publisher in English.
* "Cor" and "Amd" appear like "Cor.1" and "Amd.1" (separated by full stop) in French. In English they are "Cor 1", "Amd 1" (separated by space).